src/test/java/com/github/skjolberg/packing/LargestAreaFitFirstPackager3DTest.java

```diff
@@ -211,6 +211,45 @@ public void testStacking3xLP() {
 		assertNotNull(fits2);
 
 	}
+
+	@Test
+	public void testWith5Packets3D() {
+		List<Box> containers = new ArrayList<>();
+		containers.add(new Box(96,96,118));
+		LargestAreaFitFirstPackager packager = new LargestAreaFitFirstPackager(containers);
+
+		List<BoxItem> products1 = new ArrayList<>();
+
+		products1.add(new BoxItem(new Box("A", 45, 46, 55), 1));
```

Comment:
This will be selected as the first box, rotated with the largests area downwards (46, 55) downwards. This leaves (50, 41) for the rest of the boxes at height 45. So yeah, LAFF completely breaks down in this scenario.

Reply:
Or rather one of the (46, 46, 55) boxes will be first, but the problem is the same; it really it not very intelligent to how stacking is done. Perhaps some kind of aspect ratio perspective would be possbile to consider; at least as a quick win - i.e. cheap to calculate.